pkgs/development/interpreters/janet/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  name = "janet-${version}";
+
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "janet-lang";
+    repo = "janet";
+    rev = "v${version}";
+    sha256 = "1590f1fxb6qfhf1vp2xhbvdn2jfrgipn5572cckk1ma7f13jnkpy";
+  };
+
+  buildInputs = [ ];
+
+  JANET_BUILD=''\"v${version}\"'';
+
+  buildPhase = ''
+    PREFIX="$out" make
+  '';
+
+  installPhase = ''
+    mkdir "$out"
+    PREFIX="$out" make install
+  '';
+
+  meta = with stdenv.lib; {
```

Comment:
Need to set `meta.platfroms`, `meta.maintainers`.

Reply:
Should I just add myself to the maintainer list?

Reply:
Yes, if you plan to use it and maintain it further.

Reply:
Ok, done.